sql_user_check/README.md

```diff
@@ -0,0 +1,17 @@
+Please run each of the .sql files on the User and Database that you need to connect to LightBeam.
+Refer the examples below.
+
+Fetch Schema List:
+psql -h <DATABASE_HOST_IP> -U <DATABASE_USER> -f schema_list.sql -p <PORT> -d <DATABASE_NAME> > schema_list
```

Comment:
Use the sql markdown syntax so that README.md can display it as a sql query instead. Ditto below as well.